uxarray/grid/geometry.py

```diff
@@ -1336,3 +1337,34 @@ def _construct_boundary_edge_indices(edge_face_connectivity):
     # If an edge only has one face saddling it than the mesh has holes in it
     edge_with_holes = np.where(edge_face_connectivity[:, 1] == INT_FILL_VALUE)[0]
     return edge_with_holes
+
+
+# @njit(cache=True)
```

Comment:
Did you comment this out when debugging?

Reply:
Oh, yeah I did.